hazelcast/src/main/java/com/hazelcast/nio/serialization/SerializationServiceImpl.java

```diff
@@ -252,7 +251,7 @@ protected final int calculatePartitionHash(Object obj, PartitioningStrategy stra
         return partitionHash;
     }
 
-    public final <T> T toObject(final Object object) {
+    public final <T> T toObject2(final Object object) {
```

Comment:
Ahhh... ok.. Need to remove that..  I was switching between the 2 implementations.